isis/src/base/apps/pds2isis/pds2isis.xml

```diff
@@ -8,54 +8,69 @@ xsi:noNamespaceSchemaLocation=
-      the user may need to modify the default values in order to obtain an image 
+
+      The default settings normally work for most PDS products, but occasionally
+      the user may need to modify the default values in order to obtain an image
       that meets their specific requirements.
      </p>
      <p>
-      The incoming <def link="Special Pixels">special pixel</def> values in the 
-      PDS file are assigned equivalent values for special pixels in ISIS3.  The 
-      order of precedence for special pixels in ISIS from highest to lowest 
-      priority is the following:  
+      The incoming <def link="Special Pixels">special pixel</def> values in the
+      PDS file are assigned equivalent values for special pixels in ISIS3.  The
+      order of precedence for special pixels in ISIS from highest to lowest
+      priority is the following:
       <blockquote>
       <ol>
-        <li><def>NULL</def></li> 
-	<li><def>HRS</def></li> 
-	<li><def>LRS</def></li> 
+        <li><def>NULL</def></li>
+	<li><def>HRS</def></li>
+	<li><def>LRS</def></li>
 	<li><def>HIS</def></li>
 	<li><def>LIS</def></li>
       </ol>
-      </blockquote> 
-      If any incoming <def link="Digital Number">pixel value</def> falls within 
-      two different special pixel types, the <def link="Special Pixel">special 
-      pixel</def> with the higher priority is assigned.  The user may also 
+      </blockquote>
+      If any incoming <def link="Digital Number">pixel value</def> falls within
+      two different special pixel types, the <def link="Special Pixel">special
+      pixel</def> with the higher priority is assigned.  The user may also
       specify a range of pixel values to be assigned to the different special pixels.
-      If the range of pixel values defined by the user for the different special 
-      pixels overlap, the special pixel with the highest priority is assigned.  For 
-      example, if NULLMIN=0.0, NULLMAX=3.0, LISMIN=3.0, and LISMAX=5.0, the actual 
-      raw value 3.0 can be assigned as a NULL or LIS, but is translated to NULL because 
-      NULL has a higher priority than LIS.  The conversion of images from ISIS2 to 
-      ISIS3 follows the same rules as PDS images, except that the label information 
+      If the range of pixel values defined by the user for the different special
+      pixels overlap, the special pixel with the highest priority is assigned.  For
+      example, if NULLMIN=0.0, NULLMAX=3.0, LISMIN=3.0, and LISMAX=5.0, the actual
+      raw value 3.0 can be assigned as a NULL or LIS, but is translated to NULL because
+      NULL has a higher priority than LIS.
+    <p>
+      It should be noted for 8-bit images
+      that ISIS uses zero as it's default NULL value so any
+      <def link="Digital Number">pixel value</def> of zero will be convert to a
+      NULL value in the output cube. It is recommended that any 8-bit images
+      either be converted to 16-bit images or run the following:
```

Comment:
I wouldn't say that we recommend it for all 8-bit images. I would simply add that if you are working with an 8-bit image that needs 0 or 255 values represented as actual valid pixels, then use stretch.

Reply:
Agreed